creator-node/src/services/stateMachineManager/ContentNodeInfoManager.js

```diff
@@ -0,0 +1,81 @@
+const { logger: genericLogger } = require('../../logging')
+const _ = require('lodash')
+
+/**
+ * Queries to periodically keep the mapping of (Content Node endpoint -> SP ID)
+ * up to date.
+ * TODO: Make updateContentNodeChainInfo into a cron or its own queue after deciding on a reasonable interval.
```

Comment:
my bad here -- this is an old comment that's accomplished by the fetch-c-node-to-sp-id-map queue. so this can be removed